### ISSUE TITLE
Transceivers aren't automatically created when generating answer for Plan-B Offer

### ISSUE THREAD

**Issue by mashroor-ax** [pion/webrtc#1743]
### Your environment.
  - Version: *v3.0.19*
  - Tools: *Janus 0.7.2*
  - Other Information - *Golang version go1.13.7 linux/amd64, Ubuntu 18.04 (5.4.0-70-generic #78~18.04.1-Ubuntu)*

### What did you do?
Target was to connect to a Janus videoroom plugin and start receiving RTP packets. 
In this case there were only one publisher in the room. 

From here using the pion webrtc (v3.0.19) whenever a "joinandconfigure" message is sent to Janus following SDP is received. 

```
v=0
o=- 1617254340056160 1 IN IP4 18.188.231.160
s=VideoRoom 77
t=0 0
a=group:BUNDLE audio video data
a=msid-semantic: WMS janus
m=audio 9 UDP/TLS/RTP/SAVPF 111
c=IN IP4 18.188.231.160
a=sendonly
a=mid:audio
a=rtcp-mux
a=ice-ufrag:L1lB
a=ice-pwd:7qBnV/D85G5gH/C8ajwP0T
a=ice-options:trickle
a=fingerprint:sha-256 49:1D:14:A2:74:46:A0:62:12:A6:28:66:FF:C7:FF:C1:7B:23:66:6F:2F:E0:BD:7E:8D:E5:5E:0E:7F:45:65:9D
a=setup:actpass
a=rtpmap:111 opus/48000/2
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=ssrc:673801478 cname:janus
a=ssrc:673801478 msid:janus janusa0
a=ssrc:673801478 mslabel:janus
a=ssrc:673801478 label:janusa0
a=candidate:1 1 udp 2015363327 18.188.231.160 34965 typ host
a=candidate:4 1 udp 1679819007 18.188.231.160 34965 typ srflx raddr 172.31.27.101 rport 34965
a=end-of-candidates
m=video 9 UDP/TLS/RTP/SAVPF 96
c=IN IP4 18.188.231.160
a=sendonly
a=mid:video
a=rtcp-mux
a=ice-ufrag:L1lB
a=ice-pwd:7qBnV/D85G5gH/C8ajwP0T
a=ice-options:trickle
a=fingerprint:sha-256 49:1D:14:A2:74:46:A0:62:12:A6:28:66:FF:C7:FF:C1:7B:23:66:6F:2F:E0:BD:7E:8D:E5:5E:0E:7F:45:65:9D
a=setup:actpass
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 ccm fir
a=rtcp-fb:96 nack
a=rtcp-fb:96 nack pli
a=rtcp-fb:96 goog-remb
a=extmap:13 urn:3gpp:video-orientation
a=extmap:5 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
a=ssrc:3838312341 cname:janus
a=ssrc:3838312341 msid:janus janusv0
a=ssrc:3838312341 mslabel:janus
a=ssrc:3838312341 label:janusv0
a=candidate:1 1 udp 2015363327 18.188.231.160 34965 typ host
a=candidate:4 1 udp 1679819007 18.188.231.160 34965 typ srflx raddr 172.31.27.101 rport 34965
a=end-of-candidates
m=application 9 DTLS/SCTP 5000
c=IN IP4 18.188.231.160
a=sendrecv
a=sctpmap:5000 webrtc-datachannel 16
a=mid:data
a=ice-ufrag:L1lB
a=ice-pwd:7qBnV/D85G5gH/C8ajwP0T
a=ice-options:trickle
a=fingerprint:sha-256 49:1D:14:A2:74:46:A0:62:12:A6:28:66:FF:C7:FF:C1:7B:23:66:6F:2F:E0:BD:7E:8D:E5:5E:0E:7F:45:65:9D
a=setup:actpass
a=candidate:1 1 udp 2015363327 18.188.231.160 34965 typ host
a=candidate:4 1 udp 1679819007 18.188.231.160 34965 typ srflx raddr 172.31.27.101 rport 34965
a=end-of-candidates
```

Please note that the audio and video both are marked as '*sendonly*'. 

After setting this as remote description (*SetRemoteDescription*) and trying to create and answer (*CreateAnswer(nil)*) following is observed on the answer,
```
{answer v=0
o=- 239766094112244114 1617258014 IN IP4 0.0.0.0
s=-
t=0 0
a=fingerprint:sha-256 30:C3:B9:DF:7C:6E:50:11:17:ED:E5:BD:E3:DC:F2:4D:84:35:17:C3:4B:D7:9C:3D:10:D3:D6:B9:44:E9:AF:C3
a=group:BUNDLE audio video data
m=audio 9 UDP/TLS/RTP/SAVPF 111
c=IN IP4 0.0.0.0
a=setup:active
a=mid:audio
a=ice-ufrag:jzWwrOsXiSYejpVj
a=ice-pwd:fCSXAuOpTwsTlNKogVQEEwOMsaRPoUcR
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:111 opus/48000/2
a=inactive
m=video 9 UDP/TLS/RTP/SAVPF 96
c=IN IP4 0.0.0.0
a=setup:active
a=mid:video
a=ice-ufrag:jzWwrOsXiSYejpVj
a=ice-pwd:fCSXAuOpTwsTlNKogVQEEwOMsaRPoUcR
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 ccm fir
a=rtcp-fb:96 nack 
a=rtcp-fb:96 nack pli
a=rtcp-fb:96 goog-remb 
a=inactive
m=application 9 UDP/DTLS/SCTP webrtc-datachannel
c=IN IP4 0.0.0.0
a=setup:active
a=mid:data
a=sendrecv
a=sctp-port:5000
a=ice-ufrag:jzWwrOsXiSYejpVj
a=ice-pwd:fCSXAuOpTwsTlNKogVQEEwOMsaRPoUcR
 0xc0001c8c60}
```

Please note that in the answer audio and video both are marked as '*inactive*' which results in not receiving RTP data. 

A way around is found by manipulating the remote SDP before setting the remote descriptor like following,
```	
    remoteSDP := resp.Jsep["sdp"].(string)
	manipulatedSDP := strings.ReplaceAll(remoteSDP, "a=sendonly", "a=sendrecv")
```
In short if the remote SDP contains a=sendonly, in the answer it's always generating a=inactive.

### What did you expect?
The answer should have created a=recvonly or a=sendrecv

### What happened?
In this case the answer was always creating a=inactive

**Comment by mashroor-ax** [comment]
Thanks for getting back and really appreciate your feedback.
Yes, I am using a relatively older version of Janus which only supports Plan-B. But I also tried to explicitly create transceivers (just after creating new peerconnection) and that did not work either.

`	
	if _, err = peerConnection.AddTransceiverFromKind(webrtc.RTPCodecTypeAudio); err != nil {
		panic(err)
	} else if _, err = peerConnection.AddTransceiverFromKind(webrtc.RTPCodecTypeVideo); err != nil {
		panic(err)
	}
`

But I tried the video-room example with the same Janus endpoint and got the expected SDP.
The difference between the example and my code is,
- The example is trying to publish and I'm trying to receive.
Could there be any possibility that it might have something to do with creating Answers ?

**Comment by Sean-Der** [comment]
Hey @mashroor-ax 

Sorry for the delay, work/life has kept me really busy lately. Newer version of Janus do support Unified Plan, is it possible to upgrade?

I am going to remove Plan-B support entirely in `v4`. I don't have the bandwidth to make any changes to then unfortunately. https://github.com/pion/webrtc/issues/1914
